test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -471,7 +471,7 @@ UCS_TEST_P(test_ucp_wireup_1sided, stress_connect) {
     for (int i = 0; i < 30; ++i) {
         sender().connect(&receiver(), get_ep_params());
         send_recv(sender().ep(), receiver().worker(), receiver().ep(), 1,
-                  10000 / ucs::test_time_multiplier());
+                  1000 / ucs::test_time_multiplier());
```

Comment:
but we are also reducing the test time without valgrind?
how about `10000 / (ucs::test_time_multiplier()*ucs::test_time_multiplier())`